suite-native/module-settings/src/components/ApplicationSettings.tsx

```diff
@@ -9,12 +11,30 @@ import {
     RootStackParamList,
     RootStackRoutes,
 } from '@suite-native/navigation';
+import { Text } from '@suite-native/atoms';
 import { isDevelopOrDebugEnv } from '@suite-native/config';
+import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
 import { SettingsSection } from './SettingsSection';
 import { SettingsSectionItem } from './SettingsSectionItem';
 
+const devButtonStyle = prepareNativeStyle(() => ({
```

Comment:
Can we use something real instead of invisible box? For example clicking commit hash or version number 7 times?

Reply:
https://github.com/trezor/trezor-suite/pull/7962/commits/564357fd962741e9197a6de461ee8b7fb9d41960